atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1302,8 +1301,10 @@ private void dropTablesInternal(final Set<TableReference> tablesToDrop) throws E
 
                 for (TableReference table : tablesToDrop) {
                     CassandraVerifier.sanityCheckTableName(table);
-
                     if (existingTables.contains(table)) {
+                        if (client.describe_schema_versions().containsKey("UNREACHABLE")) {
```

Comment:
It should not considerably impact perf, as if there is a version mismatch that's not caused by a node being down (i.e., a schema mutation) it just continues on.